packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -92,8 +93,17 @@ function useHasTextDecorationControl( settings ) {
 	return settings?.typography?.textDecoration;
 }
 
-function TypographyToolsPanel( { ...props } ) {
-	return <ToolsPanel label={ __( 'Typography' ) } { ...props } />;
+function TypographyToolsPanel( { resetAllFilter, onChange, value, children } ) {
+	const resetAll = () => {
+		const updatedValue = resetAllFilter( value );
+		onChange( updatedValue );
+	};
+
+	return (
+		<ToolsPanel label={ __( 'Typography' ) } resetAll={ resetAll }>
```

Comment:
Is it intentional that this no longer passes in other props passed to this component (`{ ...props }`? It probably needs to support `panelId` being passed to it at least, so that the `ToolsPanel` `panelId` context can match the one used by a `ToolsPanelItem`?

Reply:
it's intentional because we don't want to pass the resetAll as is. but yeah I can restore the `panelId`. 

Ultimately, I don't like these APIs personally: panelId, resetAllFilter. I do understand after working on this PR why they're needed (mostly extensibility) but would love if we could find a better way to do that later.

Reply:
We don't need a "panelId" for the default case. Basically here it's the wrapper used for the Global Styles UI and there's no need for panelId there because there's no need for extensibility there. 

Thinking more though, I'm going to add it for "good measure" in case a person randomly passes a panelId to this component (before I get rid of the prop)